addon/components/property-group.hbs

```diff
@@ -46,13 +32,27 @@
                 @cacheConditionals={{@cacheConditionals}}
                 @show={{@show}}/>
         {{#if @last}}
-          {{#unless (eq index (sub this.children.length 1))}}
+          {{#if (not (eq index (sub this.children.length 1)))}}
```

Comment:
Just a small detail but maybe we should use the not-eq helper instead here?
`{{#if (not-eq index (sub this.children.length 1))}}`